man/fido_assert_new.3

```diff
@@ -15,13 +15,15 @@
 .Nm fido_assert_user_name ,
 .Nm fido_assert_authdata_ptr ,
 .Nm fido_assert_clientdata_hash_ptr ,
+.Nm fido_assert_blob_ptr ,
 .Nm fido_assert_hmac_secret_ptr ,
 .Nm fido_assert_largeblob_key_ptr ,
 .Nm fido_assert_user_id_ptr ,
 .Nm fido_assert_sig_ptr ,
 .Nm fido_assert_id_ptr ,
 .Nm fido_assert_authdata_len ,
 .Nm fido_assert_clientdata_hash_len ,
+.Nm fido_assert_blob_len ,
```

Comment:
sort

Reply:
fixed in 105034ed49b1f6c2c2b8c3e32b770366a911625f